packages/transform-dataresource/package.json

```
@@ -44,8 +44,8 @@
     "react-hot-loader": "^4.1.2",
     "react-table": "^6.8.6",
     "react-table-hoc-fixed-columns": "1.0.2",
-    "semiotic": "^1.14.4",
     "styled-jsx": "^3.1.0",
```

include this as a dependency

```suggestion
    "styled-jsx": "^3.1.0",
    "@nteract/styled-blueprintjsx": "^1.0.0",
```